zazu/repo/commands.py

```diff
@@ -8,11 +10,16 @@
 __copyright__ = "Copyright 2016"
 
 
+REPO_CHECK_EXCLUSIONS = ['clone', 'setup', 'cleanup']
```

Comment:
I wasn't clear before, setup and cleanup need the check. It would be easier to just call ctx.obj.check_repo() in those functions and remove the check in this fn.